### PR TITLE
Add getters for funded feeds & query ID from feed ID

### PR DESCRIPTION
**Pull request by oraclown**
Had to add a mapping of feed IDs to query IDs (`queryIdFromDataFeedId`), not sure if there's a better approach. Feedback wanted!

Also tried to make the README and some function descriptions a bit more clear.

Related issue: #26

### CODE REVIEW

File: contracts/Autopay.sol [hunk 1]
```diff
@@ -440,6 +470,19 @@ contract Autopay is UsingTellor {
             _rewardAmount = _feed.details.balance;
             _feed.details.balance = 0;
         }
+        // Adjust currently funded feeds
+        if (_feed.details.balance == 0) {
```
Comment (themandalore) [inline contracts/Autopay.sol:474]: I think this should be if it's less than the payment amount no?

File: contracts/Autopay.sol [hunk 2]
```diff
@@ -156,6 +160,11 @@ contract Autopay is UsingTellor {
         FeedDetails storage _feed = dataFeed[_queryId][_feedId].details;
         require(_feed.reward > 0, "feed not set up");
         _feed.balance += _amount;
+        // Add to array of feeds with funding
+        if (_feed.feedsWithFundingIndex == 0) {
```
[the remark this answers is not in the document]
Comment (oraclown) [reply contracts/Autopay.sol]: I added `&& _feed.balance >= _feed.reward` to prevent this. Does it look ok now?

File: contracts/Autopay.sol [hunk 3]
```diff
@@ -216,14 +225,17 @@ contract Autopay is UsingTellor {
         _feed.interval = _interval;
         _feed.window = _window;
         _feed.priceThreshold = _priceThreshold;
+        _feed.feedsWithFundingIndex = 0;
```
Comment (themandalore) [inline contracts/Autopay.sol:228]: you don't have to initialize it to zero, can take this line out

Comment (oraclown) [reply contracts/Autopay.sol]: Removed

File: contracts/Autopay.sol [hunk 4]
```diff
@@ -440,6 +469,18 @@ contract Autopay is UsingTellor {
             _rewardAmount = _feed.details.balance;
             _feed.details.balance = 0;
         }
+        // Adjust currently funded feeds
+        if (_feed.details.balance < _feed.details.reward) {
```
Comment (tkernell) [inline contracts/Autopay.sol:473]: we could save on gas by changing line 465 to `if (_feed.details.balance > _feed.details.reward) {`

and moving the code from lines 474-482 to the else statement on line 468

**Pull request merged**